### ISSUE TITLE
baremetal_core_console_history.state doesn't handle 'GETTING-HISTORY'

### ISSUE THREAD

**Issue by josephholsten** [oracle/terraform-provider-baremetal#153]
## Description of the bug:

```
* baremetal_core_console_history.t: unexpected state 'GETTING-HISTORY', wanted target 'SUCCEEDED'. last error: %!s(<nil>)
```

## Reproduction

`go test -v -timeout 120m -run TestCoreInstanceConsoleHistoriesDatasource`

**Comment by craigcarl-oracle** [comment]
@josephholsten Console history isn't in scope for TF. Are you trying to add it?

**Comment by josephholsten** [comment]
@craigcarl-oracle oh, I was just trying to get the existing acceptance tests to pass.

**Comment by codycushing** [comment]
Might be included in PR #159

**Comment by codycushing** [comment]
fixed in #221
